app/vmctl/testdata/servers_integration_test/remote_write_server.go

```diff
@@ -307,3 +346,19 @@ func generateTimeStampsAndValues(idx int, startTime, endTime, numOfSamples int64
 
 	return timestamps, values
 }
+
+func getTenants() map[int]struct{} {
+	min := 1000
+	max := 1002
+	d := max - min
+
+	uniqRand := make(map[int]struct{}, d)
+	for i := 0; i < d; i++ {
+		v := rand.Intn(max-min) + min
+		if _, ok := uniqRand[v]; !ok {
+			uniqRand[v] = struct{}{}
+		}
+	}
+
+	return uniqRand
+}
```

Comment:
Could you please elaborate more on this code? Why couldn't it be just `[]int{1000,1001,1002}` ?

Reply:
I removed this part of the code because all tests will do in a separate commit. 
I used this function just to generate random tenants  IDs. If it ok used only a slice of the tenants I would use it.